app/models/pregnancy.rb

```diff
@@ -7,9 +7,9 @@ class Case
 
 	field :line, type: String #DC, MD, VA
 	field :language, type: String
-	field :case_id, type: String
+	field :p_case_id, type: String
```

Comment:
@colinxfleming either is fine. Just throwing out the idea. whatever you think is better for consistency.